hw/xbox/nv2a/gl/Makefile.objs

```diff
@@ -1,4 +1,8 @@
 obj-y += gloffscreen_common.o
+obj-$(CONFIG_LINUX) += gloffscreen_glx.o
```

Comment:
nit: Why was this re-orderered? Avoid touching unaffected lines.

This also leads to a weird ordering of:
- Linux
- Windows
- macOS
- BSD

which is neither alphabetical, or by popularity.

Reply:
ok